notes/Chapter-Repeated/chap9.tex

```diff
@@ -16,373 +16,477 @@ \chapter{Repeated Games}
 Chapter based on \cite[pages 308 - 331]{MyGTAO}, \cite[Section 6.1]{ShLeMSAG}.}\\
 }
 
-A repeated game is a strategic situation where a set of players play a same game several times in a row. These situations occur quite often in real life, and in fact, most of our every-day interactions can be seen as part of a repeated game.\\
-Interestingly, the behavior of a player in a repeated game may be different than that of the same player when playing the game only once. For example, while we know that the only Nash Equilibrium of
-the Prisoner Dilemma game is for both players to betray the other, cooperation may occur rationally in the repeated game setting.
-
-The chapter is split in three sections, presenting matters from the more particular to the more general. Section \ref{sec:FinRep} discusses finitely repeated game, that can be studied directly with the analysis tools of previous chapters. Section \ref{sec:InfRep} discusses infinitely repeated games. There, we start observing interesting phenomena, in particular concerning the Prisoner's Dilemma. Finally, in Section \ref{sec:GenMod}, we present a more general and involved model for repeated games, and discuss the notions of equilibria in more details.
+A repeated game is a strategic situation where a set of players play a same
+game several times in a row. These situations occur quite often in real life
```

Comment:
I guess a space will be added here before the comma. You need to do `real life%` or `real life,`